backend/daemon/daemon_e2e_test.go

```diff
@@ -683,9 +762,10 @@ func TestMultiDevice(t *testing.T) {
 		Addrs: getAddrs(t, alice2),
 	})
 	require.NoError(t, err)
-
+	time.Sleep(100 * time.Millisecond) // to give time to trust own account
```

Comment:
Something happened with my previous comments about sleeps. It disappeared, and I can't get back to it. I still can see the sleep there, but trying to remove it the tests seem to pass fine. Can you confirm if they are still needed, and if so, why they are needed?

Reply:
I don't know, they keep coming up after rebasing. Will check that I finally delete them. Totally unuseful.